app/build.gradle

```
@@ -55,7 +55,7 @@ android {
 }
 
 dependencies {
-    def coil_version = '0.13.0'
+    def coil_version = '1.1.0'
```

Please bump to `1.1.1` .
ref: https://github.com/coil-kt/coil/releases/tag/1.1.1

```suggestion
    def coil_version = '1.1.1'
```

---

Sorry for missing v1.1.1.
I changed and committed.